packages/remark-mdx/index.js

```diff
@@ -1,10 +1,8 @@
-const unified = require('unified')
-const toMDAST = require('remark-parse')
-
 const IMPORT_REGEX = /^import/
 const EXPORT_REGEX = /^export/
 const EXPORT_DEFAULT_REGEX = /^export default/
-const BLOCKS_REGEX = '[a-z\\.]+(\\.){0,1}[a-z\\.]'
+// eslint-disable-next-line
+// const BLOCKS_REGEX = '[a-z\\.]+(\\.){0,1}[a-z\\.]'
```

Comment:
It is an unused variable, but commenting out is preferable.

Reply:
I suggest removing the `eslint-disable-next-line` comment, since you're commenting out the unused variable anyway.